imagecraft/ubuntu_image.py

```diff
@@ -106,3 +120,12 @@ def ubuntu_image_pack(
             details=details,
             resolution=resolution,
         ) from err
+
+
+def list_image_paths(workdir_path: str) -> list[str]:
+    """Extract the list of images from a ubuntu-image.json file."""
+    p = pathlib.Path(workdir_path) / "ubuntu-image.json"
```

Comment:
I also agree this is not ideal and is also a "hack". My first idea was to have ubuntu-image generate a JSON (or any other easily parse-able format) "report" that would contain the names of built artifacts. But this is very specific to the "pack" use case because otherwise the user already specifies it in the ubuntu-image.yaml.